SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -470,3 +494,13 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	// will check if there's a snapshot for them, and if so launch with it instead of the default snapshot.
 	static TMap<FString, FString> TakenSnapshots;
 };
+
+template <class T>
+UFUNCTION(BlueprintPure, Category = "Spatial Functional Test")
+T* ASpatialFunctionalTest::SpawnActor(const FActorSpawnParameters& SpawnParameters)
```

Comment:
hmm, we may want `bAutoRegister` == true by default at some point, maybe.
But it's also not hard to just hand-write the implementation for yourself at that point, leaving the decision up to you

Reply:
added it in as an option
```
	template <class T>
	T* SpawnActor(const FActorSpawnParameters& SpawnParameters = FActorSpawnParameters(), const bool bRegisterAsAutoDestroy = true);
```